package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@bcgov/citz-imb-sso-react",
-  "version": "1.0.0-alpha5",
+  "version": "1.0.0-alpha6",
```

Comment:
getting the following error when trying to install alpha6:
``` 
~/Documents/code/citz-imb-sso-react  (mvp-224-refresh-expiry-issue) $ npm i 
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: @bcgov/citz-imb-sso-react@1.0.0-alpha6
npm ERR! Found: react@18.2.0
npm ERR! node_modules/react
npm ERR!   react@"18.2.0" from the root project
npm ERR!   peer react@"^18.0.0" from @testing-library/react@15.0.3
npm ERR!   node_modules/@testing-library/react
npm ERR!     dev @testing-library/react@"15.0.3" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^18.3.0" from react-dom@18.3.0
npm ERR! node_modules/react-dom
npm ERR!   peer react-dom@"^18.0.0" from @testing-library/react@15.0.3
npm ERR!   node_modules/@testing-library/react
npm ERR!     dev @testing-library/react@"15.0.3" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! 
npm ERR! For a full report see:
npm ERR! /Users/tafriese/.npm/_logs/2024-04-25T22_21_25_022Z-eresolve-report.txt

npm ERR! A complete log of this run can be found in: /Users/tafriese/.npm/_logs/2024-04-25T22_21_25_022Z-debug-0.log
```

Reply:
deleting this because it isnt related to this branch's change

Reply:
lol you dont install alpha6 version. I'm updating it here so we can release the new version after this is merged